### DISCUSSION TITLE
Add source for Bristol city council waste collection

### DISCUSSION THREAD

**Discussion by Themaxsti** [mampfes/hacs_waste_collection_schedule#739] [Ideas]
Hi

could a source for Bristol city council waste collection be added to this.  

https://www.bristol.gov.uk/residents/bins-and-recycling/bins-and-recycling-collection-dates

not sure now easy it would be but would be very useful if it was.  

Cheers
Rob

**Comment by 5ila5** [comment]
added by #817
